data/db/src/commonMain/kotlin/io/github/droidkaigi/feeder/data/TimetableItemDao.kt

```diff
@@ -73,6 +78,18 @@ private fun TimetableItemSessionQueries.insert(session: TimetableItem.Session) {
             enTitle = session.title.enTitle,
             startsAt = session.startsAt.toEpochMilliseconds(),
             endsAt = session.endsAt.toEpochMilliseconds(),
+            jaCategory = session.category.title.jaTitle,
+            enCategory = session.category.title.enTitle,
+            targetAudience = session.targetAudience,
+            language = session.language,
+            assetSlideUrl = session.asset.slideUrl,
+            assetVideoUrl = session.asset.videoUrl,
+            jaRoomName = session.room.name.jaTitle,
+            enRoomName = session.room.name.enTitle,
+            levels = session.levels.joinToString(stringListDivider),
```

Comment:
https://cashapp.github.io/sqldelight/jvm_sqlite/types/

We can use `List` in `.sq` file, but we must add `ColumnAdapter<List<String>, String>` object to every Database Instance.
So, use `joinToString` and `split` method.